autodocsumm/__init__.py

```diff
@@ -188,7 +184,7 @@ def get_grouped_documenters(self, all_members=False):
         registry = get_documenters(self.env.app)
 
         for (mname, member, isattr) in self.filter_members(members, want_all):
-            classes = [cls for cls in six.itervalues(registry)
+            classes = [cls for cls in registry.values():
```

Comment:
Typo, missing a `]`

Reply:
aah! thanks :see_no_evil: 
```suggestion
            classes = [cls for cls in registry.values()]:
```